profiles/apparmor.d/opt.openqa-trigger-from-obs.script.rsync.sh

```diff
@@ -48,13 +48,14 @@
   /usr/bin/wc ix,
   /var/lib/openqa/osc-plugin-factory/factory-package-news/factory-package-news.py rUx,
   /usr/share/openqa/script/client rPx -> /opt/openqa-trigger-from-obs/script/rsync.sh//openqa_client,
+  /usr/bin/openqa-clirPx -> /opt/openqa-trigger-from-obs/script/rsync.sh//openqa_cli,
```

Comment:
is it missing a space between `openqa-cli` and `rPx`?

Reply:
thx, fixed